### PR TITLE
Team Section Added and Corrections in Navbar done

### PR DESCRIPTION
**Pull request by SaurabhAgarwala**
Team Section with Dummy Images added. Awaiting access rights to actual images. Hence solving #24. 

Also did minor corrections in top most navigation bar.

Screenshots of the same:

screenshot 491
screenshot 490
screenshot 489

### CODE REVIEW

**Comment by SaurabhAgarwala** [comment]
Actual Images Updated.
Screenshots of the same:
screenshot 495
screenshot 494
screenshot 493

File: index.html [hunk 1]
```diff
@@ -917,6 +921,128 @@ <h2 class="section-heading text-white">Why Partner With Us</h2>
...
+            <br>+91 94455 49535
+          </p>
+        </div>
+      </div>
+    </div>
+</div>
+
+<div class="col-xl-3 col-md-4 col-sm-6 col-12">
+  <div class="speaker-card">
+    <img class="speaker-card-img-top" src="img/Mohita Chowdhury.jpeg" alt="Mohita Chowdhury">
```
Comment (pvgupta24) [inline index.html]: Its advisable to remove spaces in `src`. Please rename the files accordingly.

**Review by pvgupta24 — CHANGES_REQUESTED** [review]
Looks good to me. Just a minor change required.
@anumehaagrawal Please merge after that.

**Comment by anumehaagrawal** [comment]
@SaurabhAgarwala  please make the necessary changes

**Comment by SaurabhAgarwala** [comment]
White spaces from image names have been removed and Anumeha's picture updated.
@anumehaagrawal @pvgupta24 Please review it now.

**Pull request merged**